<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FtpClientWindowClass</class>
 <widget class="QDialog" name="FtpClientWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ftpClient</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>472</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTreeWidget" name="fileList">
      <attribute name="headerDefaultSectionSize">
       <number>100</number>
      </attribute>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Size</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Owner</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Group</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Time</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="2" column="2">
       <widget class="QLabel" name="passWordLab">
        <property name="text">
         <string>Pa&amp;wd:</string>
        </property>
        <property name="buddy">
         <cstring>passWordlineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="ftpServerPortLab">
        <property name="text">
         <string>&amp;Port:</string>
        </property>
        <property name="buddy">
         <cstring>ftpServerPortLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="ftpServerLab">
        <property name="text">
         <string>&amp;Server:</string>
        </property>
        <property name="buddy">
         <cstring>ftpServerLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="userNameLab">
        <property name="text">
         <string>&amp;User:</string>
        </property>
        <property name="buddy">
         <cstring>userNamelineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="statusLab">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="userNamelineEdit">
        <property name="text">
         <string>Hlin</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="ftpServerLineEdit">
        <property name="text">
         <string>192.168.0.8</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="ftpServerPortLineEdit">
        <property name="text">
         <string>2121</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="passWordlineEdit">
        <property name="text">
         <string>anonymous</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="encoding">
        <property name="font">
         <font>
          <family>AlArabiya</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'AlArabiya'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;&amp;amp;Code:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="buddy">
         <cstring>ftpServerCode</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="ftpServerCode">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>utf-8</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="connectPB">
        <property name="text">
         <string>&amp;Connect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cdUpDirPB">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="ftp.qrc">
          <normaloff>:/images/cdtoparent.png</normaloff>:/images/cdtoparent.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="downLoadPB">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&amp;Download</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="upload">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&amp;Upload</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="quitPB">
        <property name="text">
         <string>&amp;Quit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>ftpServerLineEdit</tabstop>
  <tabstop>ftpServerPortLineEdit</tabstop>
  <tabstop>userNamelineEdit</tabstop>
  <tabstop>passWordlineEdit</tabstop>
  <tabstop>connectPB</tabstop>
  <tabstop>cdUpDirPB</tabstop>
  <tabstop>downLoadPB</tabstop>
  <tabstop>quitPB</tabstop>
 </tabstops>
 <resources>
  <include location="ftp.qrc"/>
 </resources>
 <connections/>
</ui>
